cluster/providers/openstack/setup.sh

```diff
@@ -26,3 +26,4 @@ source cluster/providers/openstack/utils.sh
 run_command_deployment fix_nova_mount
 run_command_deployment packstack_update_endpoints
 run_command_deployment healthcheck
+run_command_deployment packstack_patch_snapshots_support
```

Comment:
@bennyz we need to remember that.. if backup is enabled, maybe that's a good justification for using 'backup'?..

Reply:
I have a feeling backup will have the same issue, but we need to check